src/seedu/addressbook/data/person/Person.java

```diff
@@ -34,6 +34,19 @@ public Person(ReadOnlyPerson source) {
         this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getTags());
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables) {
+        String result = "";
+
+        for (Printable printable : printables) {
+            result += printable.getPrintableString();
```

Comment:
:bulb: *Java tip*: Java 8 has a `StringJoiner` class that can join multiple strings using a delimiter e.g. `aaa` `bbb` `ccc` --> `aaa, bbb, ccc` https://www.mkyong.com/java8/java-8-stringjoiner-example/

To consider: What's wrong with using string concatenation in loops?
Answer: http://www.yoda.arachsys.com/java/strings.html

Reply:
Thanks for telling me this. I have updated my code with StringJoiner.